web3/parity.py

```diff
@@ -18,6 +18,44 @@
     sign,
     unlockAccount,
 )
+from web3.shh import (
+    addPrivateKey,
+    addSymKey,
+    deleteKey,
+    deleteMessageFilter,
+    getFilterMessages,
+    getPrivateKey,
+    getPublicKey,
+    getSymKey,
+    info,
+    newKeyPair,
+    newMessageFilter,
+    newSymKey,
+    post,
+    subscribe,
+    unsubscribe,
+)
+
+
+class ParityShh(ModuleV2):
+    """
+    https://wiki.parity.io/JSONRPC-shh-module
+    """
+    info = info()
```

Comment:
It might be clearer if these methods were just assigned to variable names, rather than the output of these functions.  `info = Method(...)` and then class attribute assignment could lose the parens.  `info = info`.